docs/pages/cloud/faq.mdx

```diff
@@ -150,6 +154,10 @@ Proxy recording mode is disabled for cloud customers
 
 The ability to download recordings for offline viewing will be available in a future release.
 
+## Is there a way to forward Teleport Cloud audit events to my companies internal SEIM?
```

Comment:
I think this should be `company's`.  And probably `SIEM` should be spelled out too along with the acronym.

Reply:
Changed both of these in f0a79a209cb3678b0de09d35986ae2ce740001d.  Thanks!